music/src/core/audio_utils.ts

```diff
@@ -23,20 +23,14 @@ import * as ndarray from 'ndarray';
 //@ts-ignore
 import * as resample from 'ndarray-resample';
 
-import {fetch} from '../core/compat/global';
+import { fetch, getOfflineAudioContext, isSafari } from '../core/compat/global';
 import * as logging from './logging';
 
 // Safari Webkit only supports 44.1kHz audio.
 const WEBKIT_SAMPLE_RATE = 44100;
 const SAMPLE_RATE = 16000;
-// tslint:disable-next-line:no-any
-const appeaseTsLintWindow = (window as any);
-const isSafari = appeaseTsLintWindow.webkitOfflineAudioContext as boolean;
-// tslint:disable-next-line:variable-name
-const offlineCtx = isSafari ?
-    new appeaseTsLintWindow.webkitOfflineAudioContext(
-        1, WEBKIT_SAMPLE_RATE, WEBKIT_SAMPLE_RATE) :
-    new appeaseTsLintWindow.OfflineAudioContext(1, SAMPLE_RATE, SAMPLE_RATE);
+const sampleRate = isSafari() ? WEBKIT_SAMPLE_RATE : SAMPLE_RATE;
```

Comment:
i feel we don't use this `sampleRate` anywhere else other than this constructor, and we don't use `isSafari` outside of the offlineCtx constructor, so i would maybe not export isSafari and just pass the two sample rates to getOfflineAudioContext instead

Reply:
Yeah, that makes sense

Reply:
I also updated `isSafari` and `isWorker` to just export boolean values, since exporting a truthy function is a bit dangerous.